runtime/doc/vim_diff.txt

```diff
@@ -147,13 +147,17 @@ Highlight groups:
 4. Changed features					 *nvim-features-changed*
 
 Nvim always builds with all features, in contrast to Vim which may have
-certain features removed/added at compile-time.  This is like if Vim's "HUGE"
-build was the only Vim release type (except Nvim is smaller than Vim's "HUGE"
+certain features removed/added at compile-time.  This is like if Vim's "H
+parent E"
+build was the only Vim release type (except Nvim is smaller than Vim's "H
+parent E"
```

Comment:
For some reason, every `ug` you typed in this section was replaced with `\nparent `.